pyinfra/modules/iptables.py

```diff
@@ -0,0 +1,65 @@
+# pyinfra
+# File: pyinfra/modules/iptables.py
+# Desc: manage iptables rules
+
+'''
+The iptables modules handles iptables rules
+'''
+
+from __future__ import unicode_literals
+
+from pyinfra.api import operation
+
+
+@operation
+def forward(state, host, dport, to_ip, to_port=None,
```

Comment:
I have an issue with testing when I rename the operation:
`renamed:    pyinfra/modules/iptables.py -> pyinfra/modules/iptables/nat.py`

This gives me `tests/operations/iptables.nat.forward/add_rule.json`, but the testing framework does not handle two dots in the name of the directory. I therefore get the following error:

```
### pyinfra Tests
--> Testing everything...
...........................E
======================================================================
ERROR: Failure: ValueError (too many values to unpack (expected 2))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/Users/okso/Documents/Repos/pyinfra/venv/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/okso/Documents/Repos/pyinfra/tests/test_modules.py", line 120, in <module>
    locals()[operation_name] = make_operation_tests(operation_name)
  File "/Users/okso/Documents/Repos/pyinfra/tests/test_modules.py", line 26, in make_operation_tests
    module_name, op_name = arg.split('.')
ValueError: too many values to unpack (expected 2)
```

No other module uses this depth. How would you like to solve this ?